### ISSUE TITLE
[IntelliJ][ReportedByUser] Azure Plugin for IntelliJ Error

### ISSUE THREAD

**Issue by t-rufang** [microsoft/azure-tools-for-java#3202]
IntelliJ build version: 2019.1.3 IC-191.7479.19
OS: Windows 10
JDK: JetBrains s.r.o 1.8.0_202-release
Plugin version: 3.22.0-2019.1

Repro step:
1. Link to the latest aris cluster
2. Select WebHDFS as storage type.
3. Submit the job in remote-run way.

Error message: 
```
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "name" (class com.microsoft.azure.hdinsight.spark.common.SparkSubmitResponse), not marked as ignorable (5 known properties: "state", "id", "appId", "appInfo", "log"])
 at [Source: (String)"{"id":4,"name":"aris","state":"starting","appId":null,"appInfo":{"driverLogUrl":null,"sparkUiUrl":null},"log":["stdout: ","\nstderr: ","\nYARN Diagnostics: "]}"; line: 1, column: 17] (through reference chain: com.microsoft.azure.hdinsight.spark.common.SparkSubmitResponse["name"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:823)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1153)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1589)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1567)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:294)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4013)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3004)
	at com.microsoft.azure.hdinsight.sdk.rest.ObjectConvertUtils.convertJsonToObject(ObjectConvertUtils.java:46)
	at com.microsoft.azure.hdinsight.spark.common.SparkBatchJob.createBatchJob(SparkBatchJob.java:394)
	at com.microsoft.azure.hdinsight.spark.common.SparkBatchJob.lambda$submit$38(SparkBatchJob.java:1086)
	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:48)
	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:33)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.call(OperatorSubscribeOn.java:100)
	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
	at rx.internal.schedulers.ExecutorScheduler$ExecutorSchedulerWorker.run(ExecutorScheduler.java:107)
	at com.microsoft.intellij.rxjava.IdeaSchedulers$1.run(IdeaSchedulers.java:51)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:731)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:586)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:86)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:403)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:311)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by t-rufang** [comment]
Fixed in #3205

**Comment by v-jiche** [comment]
Different error in build azure-toolkit-for-intellij-2019.1.develop.1154.06-28-2019.
Please refer to bug: #3208
